fastlane/metadata/android/ta-IN/full_description.txt

```diff
@@ -0,0 +1,24 @@
+கடை அல்லது இணையக் கடை கட்டணத்தின் போது பிளாஸ்டிக் பரிசு அட்டைகளைத் தேடும் தேடலை நிறுத்துங்கள்.
+
+<b>கேமராவை பயன்படுத்தி பார்கோடுகளை உங்கள் சாதனத்தில் ஸ்கேன் செய்யுங்கள், அட்டைகளை மறந்து விடுங்கள்.</b>
+
+உங்கள் பணப்பையை மறந்து விடுங்கள், அல்லது அதை முக்கியமான பொருட்களுக்கு மட்டும் இலகுவாக வைத்திருங்கள்.
+
+இந்த அத்தியாவசிய தினசரி பயன்பாட்டு (EDC) கருவி மூலம் பயனற்ற பிளாஸ்டிக்குகளை பணமாக மாற்றலாம்.
```

Comment:
Is "EDC" a term used in Tamil? In English it is short for "every day carry" but I doubt it's used much in other places?

@Rajasree2004